src/pages/warnings/WarningList.tsx

```diff
@@ -163,7 +210,7 @@ const WarningList: FC = () => {
       <NotificationRow />
       <Row>
         <ScrollableTable
-          dependencies={[warnings]}
+          dependencies={[filteredWarnings]}
```

Comment:
We must use the `filteredWarnings` instead of `row` to construct the `filteredNames` parameter for this component. The undo is not correct and leaves wrong messaging on selecting warnings with active filters. My point was we should only change the variable, not the map logic applied to it.

Reply:
Ah never mind, this is correct as is.